test/support/test_utils_grpc.ex

```diff
@@ -213,4 +234,131 @@ defmodule Logflare.TestUtilsGrpc do
       value: %AnyValue{value: {:double_value, :rand.uniform(100) + 1.00}}
     }
   end
+
```

Comment:
would be good if we can eventually migrate this to use StreamData for generation for better property testing. not a priority